content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
+
+![EEW Logo](./EEW-logo.png)
 
-* Draw large-scale public attention to lack of environmental enforcement.
+## Our Goals
+* Draw large-scale public attention to the lack of environmental enforcement.
 * Work with partner organizations, especially marginalized communities, to build data efficacy through mutually beneficial relationships.
-* Inform EPA visions of alternative forms of public engagement—a Green New Deal for environmental data.
-* Prototype community-oriented structures for environmental data, i.e. by health factors for a community.
+* Inform EPA visions of alternative forms of public engagement
+* Prototype community-oriented structures for environmental data, i.e. by congressional district, watershed, and health risks for a community.
 * Bring attention to the relationship between environmental enforcement, environmental racism, and other interlocking forms of oppression (particularly white supremacy, capitalism, settler-colonialism, heteropatriarchy, etc.)
 
 ![Environmental Data and Governance Initiative logo](./EDGI-logo.png)
 
 ## The Environmental Data and Governance Initiative
+EDGI is an international network of over 175 members from more than 80 different academic institutions and non-profits, comprised primarily of grassroots volunteer efforts. Since 2016, EDGI has served as a preeminent watchdog group for federal environmental data, generating international efforts to duplicate and monitor repositories of public data vital to environmental health research and knowledge.
 
-EDGI is an international network of over 175 members from more than 80 different academic institutions and non-profits, comprised foremost by grassroots volunteer efforts. Since 2016, EDGI has served as a preeminent watchdog group for federal environmental data, generating international effort to duplicate and monitor repositories of public data that are vital to environmental health research and knowledge.
-
-EDGI’s work has been widely acknowledged, leading to EDGI testifying before Congress on declines in EPA enforcement, and hundreds of mentions in leading national and international media such as The New York Times, The Washington Post, Vice News, and CNN. For more about our work, read our <a href="https://envirodatagov.org/publication/edgi-annual-report-2019/" target=_blank >2019 Annual Report</a> and <a href="https://envirodatagov.org/publication/edgi-annual-report-2020/" target=_blank >2020 Annual Report</a>.
+EDGI’s work has been widely acknowledged, leading to EDGI members testifying before Congress on declines in EPA enforcement, and hundreds of mentions in leading national and international media such as The New York Times, The Washington Post, Vice News, and CNN. For more about our work, read our [2019 Annual Report](https://envirodatagov.org/publication/edgi-annual-report-2019/) and [2018 Annual Report](https://envirodatagov.org/publication/edgi-annual-report-2018/). 
```

Comment:
These should be changed to 2019 and 2020 annual reports https://envirodatagov.org/publications/